app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    params = command.split(" ")
+    command_name = params[0]
+    file_name = params[1]
+    file_name_copy = params[2]
```

Comment:
You can use unpacking here, but first of all check if there is 3 params in command